src/migrations/20211202212104-convert-grantee-to-recipient.js

```diff
@@ -0,0 +1,172 @@
+
+      /*
+      * Grantee actions:
+      * rename table to 'Recipients'
+      * rename 'granteeType' to 'recipientType'
+      */
+      await queryInterface.renameTable('Grantees', 'Recipients', { transaction });
+      await queryInterface.renameColumn('Recipients', 'granteeType', 'recipientType', { transaction });
+
+      /*
+      * Activity Report actions:
+      * update 'activityRecipientType'
+      *   'grantee' to 'recipient'
+      *   'non-grantee' to 'other-entities'
+      *
+      * update legacy reports 'activityRecipientType'
+      *   'imported' field has 'nonGranteeActivity' value to 'other-entities'
+      *   'imported' field has empty 'nonGranteeActivity' to 'recipient'
+      *
+      * update 'reason' of 'New Grantee' to 'New Recipient'
+      *
+      * update 'requested by'
+      *   'Regional Office' to 'regionalOffice' (legacy reports)
+      *   'Grantee' to 'recipient'              (legacy reports)
+      *   'grantee' to 'recipient'
+      */
+      await queryInterface.sequelize.query('UPDATE "ActivityReports" SET "activityRecipientType" = \'recipient\' WHERE "activityRecipientType" = \'grantee\';', { transaction });
+      await queryInterface.sequelize.query('UPDATE "ActivityReports" SET "activityRecipientType" = \'other-entities\' WHERE "activityRecipientType" = \'non-grantee\';', { transaction });
+
+      await queryInterface.sequelize.query('UPDATE "ActivityReports" SET "activityRecipientType" = \'recipient\' WHERE "activityRecipientType" = \'nonGrantee\' and imported->>\'nonGranteeActivity\' = \'\';', { transaction });
+      await queryInterface.sequelize.query('UPDATE "ActivityReports" SET "activityRecipientType" = \'other-entities\' WHERE "activityRecipientType" = \'nonGrantee\' and imported->>\'nonGranteeActivity\' != \'\';', { transaction });
+
+      await queryInterface.sequelize.query('UPDATE "ActivityReports" SET "reason" = ARRAY_REPLACE("reason", \'New Grantee\', \'New Recipient\');', { transaction });
+
+      await queryInterface.sequelize.query('UPDATE "ActivityReports" SET "requester" = \'regionalOffice\' WHERE "requester" = \'Regional Office\';', { transaction });
+      await queryInterface.sequelize.query('UPDATE "ActivityReports" SET "requester" = \'recipient\' WHERE "requester" = \'Grantee\';', { transaction });
+      await queryInterface.sequelize.query('UPDATE "ActivityReports" SET "requester" = \'recipient\' WHERE "requester" = \'grantee\';', { transaction });
+
+      /*
+      * GrantGoals actions:
+      * rename 'granteeId' to 'recipientId'
+      * update foreign key 'GrantGoals_granteeId_fkey' to 'GrantGoals_recipientId_fkey'
+      */
+      await queryInterface.renameColumn('GrantGoals', 'granteeId', 'recipientId', { transaction });
+      await queryInterface.sequelize.query('ALTER TABLE "GrantGoals" RENAME CONSTRAINT "GrantGoals_granteeId_fkey" TO "GrantGoals_recipientId_fkey";', { transaction });
+    });
+  },
```

Comment:
In the down sections of migrations we have already written the way we were removing enum values is [not supported](https://www.postgresql.org/docs/current/datatype-enum.html) (see section 8.7.4). We could drop the enum type and recreate it with the values we want to use, which may be as simple as it sounds or may be more involved since it is a data type we already use, I'm not sure.